core/jvm/src/main/scala/scalaz/zio/RTS.scala

```diff
@@ -79,7 +74,7 @@ trait RTS {
    */
   val YieldMaxOpCount = 1024
 
-  private final def newFiberContext[E, A](handler: List[Throwable] => IO[Nothing, Unit]): FiberContext[E, A] = {
+  private final def newFiberContext[E, A](handler: Cause[Nothing] => IO[Nothing, Unit]): FiberContext[E, A] = {
```

Comment:
Again, I think the handlers will have `Cause[Any] => IO[Nothing, Unit]` as their type.